packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -50,11 +52,16 @@ describe("Temporal Collab Spaces", () => {
 	const cols = 7;
 	const rows = 20;
 
+	const runtimeOptions: IContainerRuntimeOptions = {
+		enableGroupedBatching: true,
+		chunkSizeInBytes: 950000,
```

Comment:
Is there a particular reason for these values?  Were they an outcome from experimentation?  Can we add some comments ?